meshroom/ui/qml/Controls/NodeActions.qml

```diff
@@ -0,0 +1,263 @@
+            onPressed:       function(mouse) { mouse.accepted = true }
+            onReleased:      function(mouse) { mouse.accepted = true }
+            onClicked:       function(mouse) { mouse.accepted = true }
+            onDoubleClicked: function(mouse) { mouse.accepted = true }
+            hoverEnabled: false
+        }
+
+        // Update position
+        function updatePosition() {
+            if (!selectedNodeDelegate || !draggable) return
+            
+            // Calculate node position in screen coordinates
+            const nodeScreenX = selectedNodeDelegate.x * draggable.scale + draggable.x
+            const nodeScreenY = selectedNodeDelegate.y * draggable.scale + draggable.y
+            
+            // Position header above the node (fixed offset in screen pixels)
+            x = nodeScreenX + (selectedNodeDelegate.width * draggable.scale - width) / 2
+            y = nodeScreenY - height - headerOffset
+        }
+
+        // Update position when the user moves on the graph
+        Connections {
+            target: root.draggable
+            function onXChanged()     { actionHeader.updatePosition() }
+            function onYChanged()     { actionHeader.updatePosition() }
+            function onScaleChanged() { actionHeader.updatePosition() }
+        }
+        
+        // Update position when nodes are moved
+        Connections {
+            target: actionHeader.selectedNodeDelegate
+            function onXChanged() { actionHeader.updatePosition() }
+            function onYChanged() { actionHeader.updatePosition() }
+            ignoreUnknownSignals: true
+        }
+        
+        // Set initial position
+        onSelectedNodeDelegateChanged: updatePosition()
+
+        // Listen to updates to status
+        Connections {
+            target: actionHeader.selectedNode
+            function onGlobalStatusChanged()   { actionHeader.currentStatus = target.globalStatus }
+            function onGlobalExecModeChanged() { actionHeader.currentExecMode = target.globalExecMode }
+            function onIsComputedChanged()     { actionHeader.nodeCanBeStopped = target.canBeStopped() }
+            function onLockedChanged()         { actionHeader.nodeLocked = target.locked }
+            ignoreUnknownSignals: true
+        }
+
+        // Also listen to uigraph for status updates
+        Connections {
+            target: root.uigraph
+            function onComputationStatusChanged() { 
+                actionHeader.currentStatus = actionHeader.selectedNode ? actionHeader.selectedNode.globalStatus : "NONE"
+            }
+            function onNodeStatusUpdated() {
+                actionHeader.currentStatus = actionHeader.selectedNode ? actionHeader.selectedNode.globalStatus : "NONE"
+            }
+            ignoreUnknownSignals: true
+        }
+
+        Row {
+            id: actionItemsRow
+            anchors.centerIn: parent
+            spacing: 2
+            
+            // Compute button
```

Comment:
I would prefer a single button for the current compute/recompute.
This button would have 3 steps:
- Compute (when not computed)
- Stop (when computing)
- Delete Data (when computed)

When computed, if you want to recompute, you just have to click twice (Delete & Compute).

I would use an orange background for Stop and a red background for Delete.